mappings/services.json

```diff
@@ -1,5 +1,8 @@
 {
   "settings": {
+    "index": {
+      "max_result_window": 20000
+    },
```

Comment:
Do we want to include the equivalent setting on the DOS mapping, so we at least have visibility of what the default is?

Reply:
Hmm. Good question.

Reply:
The answer is yes, and I've done so now